schemas/lifeCycleAssessmentData.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/building-envelope-data/api/v1.0.0/schemas/lifeCycleAssessmentData.json",
```

Comment:
You can copy the definition "data" into the definitions "$defs" of lifeCycleData.json and refer to it within lifeCycleData.json . Then each `domain` has only two files like lifeCycle.json and lifeCycleData.json . It's still modular because of the definitions.